sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyAsyncClient.java

```diff
@@ -629,14 +626,33 @@
      *   .map(Response::value);
      * </pre>
```

Comment:
@g2vinay You should get these code samples out of here and into code snippet files.

Reply:
okay, will make a PR this week.